roles/common/files/libexec/smart.sh

```diff
@@ -288,5 +306,72 @@ normal_smart ()
   done
 }
 
+nvme_smart ()
+{
+  # Loop through NVMe devices
+  for nvmedisk in $(sudo $nvmecli list | grep nvme | awk '{ print $1 }')
+  do
+    # Include NVMe devices in overall drive count
+    let "numdrives+=1"
+    # Clear output variable from any previous disk checks
+    output=""
+    output=$(sudo $nvmecli smart-log $nvmedisk | \
```

Comment:
also, did you consider smart-log-add and reject it?  I don't see that there's anything more useful there, but, just wondering if you noticed